test/unit/epilogue/warp/fragment_iterator_volta_tensor_op.cu

```diff
@@ -32,6 +32,8 @@
     \brief Unit tests for thread-level GEMM
 */
 
+#include <fstream>
```

Comment:
This include is not necessary, as `ofstream` is commented out below

Reply:
Indeed, thanks.